apollo-router/src/plugins/coprocessor/mod.rs

```diff
@@ -77,19 +78,26 @@ impl Plugin for CoprocessorPlugin<HTTPClientService> {
             .with_native_roots()
             .with_no_client_auth();
 
-        let connector = hyper_rustls::HttpsConnectorBuilder::new()
+        let builder = hyper_rustls::HttpsConnectorBuilder::new()
             .with_tls_config(tls_config)
             .https_or_http()
-            .enable_http1()
-            .enable_http2()
-            .wrap_connector(http_connector);
+            .enable_http1();
+
+        let connector = if init.config.traffic_shaping.is_none() || init.config.traffic_shaping.as_ref().unwrap().experimental_http2 == Some(Http2Config::Enable) {
```

Comment:
Considering the case of new version delivered by customer with, wanting to keep same behavior if no change to their config yaml.... if they havent provided the newly added traffic_shaping, or if they have provided it with but have not used the disabled http2 enum value, then enable http2.